granulate_utils/metadata/databricks_client.py

```diff
@@ -46,12 +55,25 @@
 
 
 class DBXWebUIEnvWrapper:
-    def __init__(self, logger: logging.LoggerAdapter, enable_retries: bool = True) -> None:
+    def __init__(
+        self,
+        logger: logging.LoggerAdapter,
+        enable_retries: bool = True,
+        http_request_timeout: int = DEFAULT_REQUEST_TIMEOUT,
+        raise_on_failure: bool = False,
+        verify_request_ssl: bool = DEFAULT_REQUEST_VERIFY_SSL,
+    ) -> None:
         """
         When `enable_retries` is True, the wrapper will retry the request to the webui until it succeeds or until
         """
```

Comment:
The sentence here is cut off 😕

Reply:
Also needs doc-ing for the new values.

Reply:
Old @roi-granulate's code, removing the comment